roseus/roseus.cpp

```diff
@@ -1429,6 +1429,17 @@ pointer ROSEUS_HAS_PARAM(register context *ctx,int n,pointer *argv)
   return((ros::param::has(key))?(T):(NIL));
 }
 
+pointer ROSEUS_DEL_PARAM(register context *ctx,int n,pointer *argv)
+{
+  string key;
+
+  ckarg(1);
+  if (isstring(argv[0])) key.assign((char *)get_string(argv[0]));
+  else error(E_NOSTRING);
+
+  return((ros::param::del(key))?(T):(NIL));
```

Comment:
@k-okada Yes. in global namespace `ros::param::del` and also `ros::NodeHandle` has accessor by `nh.deleteParam`.